### PR TITLE
PMM-6432 Upgrade to go 1.16

### PR DESCRIPTION
**Pull request by BupycHuk**
https://github.com/Percona-Lab/pmm-submodules/pull/1773
https://github.com/percona-platform/dbaas-controller/pull/184
https://github.com/percona/pmm-managed/pull/758
https://github.com/percona-platform/dbaas-api/pull/126

### CODE REVIEW

File: build/rpmbuild-docker/Dockerfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ RUN yum -y remove nodesource-release-el7-1.noarch
 RUN yum clean all && rm -rf /var/cache/yum
 
 # keep that format for easier search
-ENV GO_VERSION go1.15.7
+ENV GO_VERSION go1.16.4
 ENV GO_CHECKSUM 0d142143794721bb63ce6c8a6180c4062bcf8ef4715e7d6d6609f3a8282629b3
```
Comment (EvgeniyPatlan) [inline build/rpmbuild-docker/Dockerfile]: go checksum must be updated too

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1773-99cfd95
client docker - perconalab/pmm-client-fb:PR-1773-99cfd95
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1773-99cfd95.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1773-99cfd95&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1773-99cfd95.tar.gz

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/8445/

File: build/rpmbuild-docker/Dockerfile (hunk 1 above)
Comment (jprukner) [reply build/rpmbuild-docker/Dockerfile]: done

**Comment by jprukner** [comment]
UI tests failing are not related to changes made.

**Review by puneet0191 — DISMISSED** [review]

File: build/bin/vars [hunk 2]
```diff
@@ -28,7 +28,7 @@ unset new_pmm_version
 unset new_pmm_release
 echo -e "\n\n\n>>> full_pmm_version=${full_pmm_version} pmm_version=${pmm_version} pmm_release=${pmm_release}\n\n\n"
 
-rpmbuild_docker_image=${RPMBUILD_DOCKER_IMAGE:-perconalab/rpmbuild:2}
+rpmbuild_docker_image=${RPMBUILD_DOCKER_IMAGE:-perconalab/rpmbuild:2-go1.16}
```
Comment (EvgeniyPatlan) [inline build/bin/vars]: Such change shouldn't go to production. we need to build rpmbuild docker image with tag 2

**Review by EvgeniyPatlan — CHANGES_REQUESTED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1773-99cfd95
client docker - perconalab/pmm-client-fb:PR-1773-99cfd95
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1773-99cfd95.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1773-99cfd95&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1773-99cfd95.tar.gz

**Review by EvgeniyPatlan — APPROVED** [review]

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1773-0762344
client docker - perconalab/pmm-client-fb:PR-1773-0762344
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1773-0762344.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1773-0762344&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1773-0762344.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-1773-145ea07
client docker - perconalab/pmm-client-fb:PR-1773-145ea07
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1773-145ea07.tar.gz
Create Staging Instance: https://pmm.cd.percona.com/job/aws-staging-start/parambuild/?DOCKER_VERSION=perconalab/pmm-server-fb:PR-1773-145ea07&CLIENT_VERSION=https://s3.us-east-2.amazonaws.com/pmm-build-cache/PR-BUILDS/pmm2-client/pmm2-client-PR-1773-145ea07.tar.gz

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/8531/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/8532/

**Comment by JNKPercona** [comment]
UI tests have failed, Please check: UI: https://pmm.cd.percona.com/job/pmm2-ui-tests/8533/

**Pull request merged**